[
  {
    "set": "asa",
    "id": 1,
    "messages": [
      {
        "msg": "usage",
        "id": 1,
        "description": "Usage instructions, {0} is the program name.",
        "C": "Usage: {0} [<file>...]"
      },
      {
        "msg": "uncaught_std_exception",
        "id": 2,
        "description": "We have an uncaught exception of type std::exception, {0} is the what.",
        "C": "Internal error: {0}."
      },
      {
        "msg": "uncaught_exception",
        "id": 3,
        "description": "We have an uncaught exception of unknown type.",
        "C": "Internal error: Unknown cause."
      }
    ]
  }
]
